### DISCUSSION TITLE
Le système solaire (Maxime L.)

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#30] [Q&A]
Commentaires sur ce quiz

**Accès**
- Serveur de développement (protégé) :  https://timunix3.csfoy.ca/~quiz-TIM/maxamax2/index.html
- Dépôt Github (privé pour le moment): https://github.com/maxamax2/quiz-tim
- Github.io :

**Comment by alexandradias** [comment]
Tout d'abord, très sympa comme thème…

Le quiz est agréable à la navigation, les interactions y sont présentes et réactives, les images bien mise en valeur.
J'aurais réduit la largeur du bloc texte en version table, afin d'accentuer l'envie à la lecture qui est très intéressante. 
(cf. la capture que je te propose).
Pour éviter les espaces entre le texte, j'aurais modifié le css en version mobile, en faisant un text-align:center, aussi bien sur la section réponse que sur le score.
Même en désactivant le javascript le quiz reste accessible. Le HTML ne comporte aucune erreur.

Tout est vraiment parfaitement réalisé, tu as répondu pour moi à toutes les attentes demandées pour ce projet.
Encore bravo!

 
<img width="1440" alt="Capture d’écran 2023-05-10 à 14 18 41" src="https://github.com/integration2/quizTIM2023/assets/52402064/c50136a8-2f83-4c92-b02c-dba66358ddae">
<img width="1440" alt="Capture d’écran 2023-05-10 à 14 20 26" src="https://github.com/integration2/quizTIM2023/assets/52402064/a84d14e5-c91b-49d9-8e67-44cd09c17793">

**Comment by evefevrier** [comment]
@maxamax2 
Une petite amélioration possible sur le click du choix de réponse, éviter que ça bouge... l'astuce pour cela est de prévoir à l'état normal du label une bordure transparente de même épaisseur que celle de l'état _checked_

**Comment by ARTADestrokzZ** [comment]
@maxamax2 
Très bon quiz, j'aime bien le thème sur notre système solaire. Je n'ai vu aucun défaut. Je suis peut-être aveugle mais sinon bravo quand même.
